README.md

```diff
@@ -55,3 +55,73 @@ You can also navigate to localhost:5000 and see a served page.
 Swagger documentation - `localhost:5000/documentation`
 
 Healthcheck monitoring - `localhost:5000/__health`
+
+## Testing
+`$ cd <cnn-hapi-root>`
+`$ npm run example-server`
+This will run the example server in the `./example` directory.
+
+## ENV VARS
```

Comment:
Could the ENV_VARs section be changed into a list, bulleted or otherwise? Previewing the README.md shows them as one jumbled paragraph

Reply:
will change now

Reply:
just updated @tombrake

Reply:
looks good 👍